clang/test/CodeGenHLSL/builtins/RWBuffer-elementtype.hlsl

```diff
@@ -2,36 +2,36 @@
 // RUN: %clang_cc1 -triple spirv-pc-vulkan-compute -finclude-default-header -fnative-half-type -emit-llvm -o - %s | FileCheck %s -check-prefixes=SPIRV
 
 // NOTE: The type name number and whether the struct is packed or not will mostly
-// likely change once subscript operators are properly implemented (llvm/llvm-project#95956) 
+// likely change once subscript operators are properly implemented (llvm/llvm-project#95956)
 // and theinterim field of the contained type is removed.
```

Comment:
Please remove :)

Reply:
Sorry I was not clear! I meant you can remove the whole comment in all of the *elementtype.hlsl tests. :)